### ISSUE TITLE
Exclusion areas show up on map in release

### ISSUE THREAD

**Issue by billw2012** [Sparker95/Vindicta#324]
Need to hide them on load.

**Comment by marvis999** [comment]
Before anyone makes a scripting solution for this: It might be possible to hide them in the editor by making them transparent

**Comment by billw2012** [comment]
Yeah that would be ideal, although we want to see them in the editor of course 😄
